outdoor_dwa.py

```diff
@@ -92,8 +93,22 @@ def __init__(self):
         self.x = 0.0
         self.y = 0.0
         self.z = 0.0
-        self.goalX = 0
-        self.goalY = 0
+
```

Comment:
Redundant code

Reply:
Actually this is just a placeholder for declaring the class variable.

Reply:
this comment is related to the commented code below this in pull request